ui/src/services/version/RouterVersion.js

```diff
@@ -4,10 +4,16 @@ import { Ensembler } from "../ensembler";
 import { get } from "../../components/form/utils";
 import { stripKeys } from "../../utils/object";
 import { Status } from "../status/Status";
+import { routingStrategyOptions } from "../../router/components/form/components/ensembler_config/standard_ensembler/typeOptions";
 
 const _ = require(`lodash`);
 const objectAssignDeep = require(`object-assign-deep`);
 
+const parseStandardEnsemblerConfig = (config) => {
```

Comment:
This function and its usage are the only changes on this file. The rest of the diffs are from formatting changes.